dev-libs/libusb/libusb-1.0.21.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="A library that provides generic access to USB devices"
+DESCRIPTION="libusb is meant to facilitate the communication with USB hardware.
+
+- portable: Using a single cross-platform API, it provides access to USB \
+devices on Linux, OS X, Windows and OpenBSD.
+- user-mode: No special privilege or elevation is required for the \
+application to communicate with a device.
+- version-agnostic: All versions of the USB protocol, from 1.0 to 3.0 \
+(latest), are supported."
+HOMEPAGE="http://libusb.info/"
+COPYRIGHT="2001 Johannes Erdfelt
+	2007-2009 Daniel Drake
+	2010-2012 Peter Stuge
+	2008-2013 Nathan Hjelm
+	2009-2013 Pete Batard
+	2009-2013 Ludovic Rousseau
+	2010-2012 Michael Plante
+	2011-2013 Hans de Goede
+	2012-2013 Martin Pieuchot
+	2012-2013 Toby Gray"
+         2013-2015 Chris Dickens"
+LICENSE="GNU LGPL v2.1"
+REVISION="2"
+SOURCE_URI="https://github.com/libusb/libusb/archive/v1.0.21.tar.gz"
```

Comment:
You might wish to use:
```
REVISION="1"
SOURCE_URI="https://github.com/libusb/libusb/archive/v$portVersion.tar.gz"
```

BTW, you should probably remove the recipe for 1.0.20.

While at it, you can also add a TEST at the end:
```
TEST()
{
	make check
}
```